main.py

```diff
@@ -1,12 +1,22 @@
 """ToDo."""
-from tasks.kyu8 import example
+from tasks.kyu8 import heads_legs
 
 
-def run_example():
-    """ToDo"""
-    first = input("a: ")
-    second = input("b: ")
-    print(example(first, second))
+def run_heads_legs():
+    """
+    Runs heads_legs function.
+    """
+    try:
+        heads = int(input("Enter the amount of heads: "))
+        legs = int(input("Enter the amount of legs: "))
+        while heads not in range(0, 1001) or legs not in range(0, 1001):
+            print("Bad input! Values must be positive and not greater than 1000.")
+            heads = int(input("Enter the amount of heads: "))
+            legs = int(input("Enter the amount of legs: "))
+    except ValueError:
+        print("Wrong input!")
+    print(heads_legs(heads, legs))
```

Comment:
Consider calling the function inside a try-block because there is a chance you will call it with incorrect parameters

Reply:
fixed.